lib/mayaUsd/fileio/translators/translatorCamera.cpp

```diff
@@ -204,13 +208,173 @@ static bool _CreateAnimCurveForPlug(
     return true;
 }
 
+#if PXR_VERSION >= 2411
+static MFnAnimCurve::InfinityType _ConvertUsdExtrapolationTypeToMaya(TsExtrapMode usdExtrapolation)
+{
```

Comment:
I won't review the details since IDK about USD splines and Maya anim curve much.